clang/include/clang-c/Dependencies.h

```diff
@@ -325,7 +325,7 @@ clang_experimental_DependencyScannerWorker_getFileDependencies_v3(
     CXModuleLookupOutputCallback *MLO, unsigned Options, CXString *error);
 
 /**
- * See \c clang_experimental_DependencyScannerWorker_getFileDependencies_v5.
+ * See \c clang_experimental_DependencyScannerWorker_getDepGraph.
```

Comment:
We're now referring to another function with very different API, which might be confusing. We should probably copy the `_v5` documentation here (adjusted for the error-reporting difference) and refer to that from the `_v3` API. WDYT?

Reply:
How about I add a comment saying this function is deprecated and `clang_experimental_DependencyScannerWorker_getDepGraph` should be used instead?

Reply:
That sounds good. But even though the API will be marked as deprecated, I think it would make sense to have it documented, until we decide to drop it entirely.

Reply:
See c5e3a78f011785f70a392339a04360f305faeb59

Reply:
Thanks!